concent_api/core/tests/utils.py

```diff
@@ -569,6 +567,32 @@ def _get_serialized_force_payment(
             settings.CONCENT_PUBLIC_KEY
         )
 
+    def _get_deserialized_subtask_results_verify(
+        self,
+        timestamp=None,
+        subtask_results_rejected=None,
+    ):
+        """ Return SubtaskResultsVerify deserialized """
+        with freeze_time(timestamp or self._get_timestamp_string()):
+            return message.concents.SubtaskResultsVerify(
+                subtask_results_rejected=(
+                    subtask_results_rejected or
+                    self._get_deserialized_subtask_results_rejected()
+                ),
+            )
+
+    def _get_serialized_subtask_results_verify(
+        self,
+        timestamp=None,
+        subtask_results_verify=None,
+        provider_private_key=None
+    ):
+        return dump(
+            subtask_results_verify or self._get_deserialized_subtask_results_verify(timestamp),
+            provider_private_key or self.PROVIDER_PRIVATE_KEY,
```

Comment:
These are not boolean values. Please compare them explicitly.

Reply:
this conventione is widely used throughout the rest of the code.

Reply:
Where ? :) We are actually trying to avoid this.

Reply:
@dybi What about this one ?

Reply:
@dybi

Reply:
in `ConcentIntegrationTestCase` class. Actually, I see we use two patterns:
```python
    def _get_serialized_force_get_task_result(
        self,
        report_computed_task,
        timestamp,
        requestor_private_key = None
    ):
        """ Returns MessageForceGetTaskResult serialized. """
        with freeze_time(timestamp):
            force_get_task_result = message.concents.ForceGetTaskResult(
                report_computed_task    = report_computed_task,
            )
        return dump(
            force_get_task_result,
            requestor_private_key or self.REQUESTOR_PRIVATE_KEY,
            settings.CONCENT_PUBLIC_KEY
        )
```
or
```python
    def _sign_message(self, golem_message, client_private_key = None):
        return sign_message(
            golem_message,
            self.REQUESTOR_PRIVATE_KEY if client_private_key is None else client_private_key,
        )
```
I am fine with both and would use the one which is more readible in given place.
@rwrzesien, what's you opinion?

Reply:
We should compare explicitly if value is not bool, please fix the place you have found too.